.github/workflows/test.yml

```
@@ -0,0 +1,15 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: "go test"
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
```

```suggestion
          go-version: 'stable'
```

This repo is infrequently updated, so let's just configure it to always run CI with the latest stable version of Go.